js/blocks/WidgetBlocks.js

```diff
@@ -1430,6 +1430,7 @@ function setupWidgetBlocks(activity) {
             logo.phraseMaker.rowArgs = [];
             logo.phraseMaker.graphicsBlocks = [];
             logo.phraseMaker.clearBlocks();
+            logo.phraseMaker._lyricsON = false;
```

Comment:
This needs to change to logo.phraseMaker.lyricsOn = false;

(Did you test after your updates?)